### DISCUSSION TITLE
How to replace srcset ?

### DISCUSSION THREAD

**Discussion by asimex** [GrapesJS/grapesjs#6201] [Q&A] [answered]
In GrapesJS, I'm importing an HTML page that contains an image with a srcset attribute. When I try to replace the image, my new image link always goes into the src attribute instead of srcset. How can I fix this? Please help.

**Comment by mohamedsalem401** [comment, marked as answer]
Hey, sorry for late response.
Please try the following snippit
```ts
editor.on('component:update:src', (component) => {
  if (component.get('tagName') !== 'img') return
  component.addAttributes({ 'srcset': component.getAttributes().src })
});
```

If you want to set the `srcset` after loading the component, just add 
```ts
editor.on('component:mount', (component) => {
  if (component.get('tagName') !== 'img') return
  component.addAttributes({ 'srcset': component.getAttributes().src })
});
```

**Reply by asimex** [reply]
great it worked thanks a lot
